pyproject.toml

```diff
@@ -35,18 +35,19 @@ documentation = "https://softwaremill.github.io/FoXAI/"
 "Bug Tracker" = "https://github.com/softwaremill/foxai/issues"
 
 [tool.poetry.dependencies]
-python = ">=3.7.2,<3.11"
+python = ">=3.8.0,<3.11"
```

Comment:
I remember we wanted to support Python 3.7 few months ago as quite some notebooks and users were using that: https://git.afpy.org/mdk/python-versions

Reply:
Agree.